app/pb_kits/playbook/pb_title/_title.jsx

```diff
@@ -2,38 +2,45 @@
 
 import React from 'react'
 import classnames from 'classnames'
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
 import { spacing } from '../utilities/spacing.js'
 
 type TitleProps = {
+  aria?: object,
   className?: String,
   children?: Array<React.ReactNode> | React.ReactNode,
   dark?: Boolean,
+  data?: object,
+  id?: String,
   size?: 1 | 2 | 3 | 4,
+  tag?: "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "p" | "div" | "span",
```

Comment:
What might the usecase be for `tag="p"`? Would we want to instead settle for `div` instead of `span` or is there a usecase for both?

Reply:
Hi, Thanks for the question. I'm not sure what the usecase would be for either; it does seem a little strange for a Title to be in a "span" or "p" tag. I just put these in to mimic the rails implementation.  

https://github.com/powerhome/playbook/blob/1f34586588b17ed9ede3a74d282807d114e5a51e/app/pb_kits/playbook/pb_title/title.rb#L15-L16

If you'd like, I can fix both sides to remove the "p" and "span" tags.

Reply:
That would be great. Thank you!